### ISSUE TITLE
Distroless Python - Support for Python 3.6+

### ISSUE THREAD

**Issue by jbmcfarlin31** [GoogleContainerTools/distroless#476]
Hi all,

I have been playing with the distroless Python containers, specifically `python3` and `python3-debian10`. I am using a mulit-stage docker build with my base image being `python:3.8-slim-buster`. I build out my virtual environment, and install the needed packages for my app to run, then copy the virtual environment over to the distroless image. However, one of my packages requires python 3.6+ to run. When I try to execute the app within the distroless container, it fails because python 3.5 is the only one loaded.

I have seen other issues within this repo regarding the same kind of content around python 3.6, and some of the responses seem the answer is to build upon a different base image like `sid` or `buster` as mentioned in this [comment here](https://github.com/GoogleContainerTools/distroless/issues/111#issuecomment-403922200). 

Though I am not entirely sure how to do that. So I guess I have two questions:
1) Is support for python 3.6 distroless images coming?

2) If no, how can I perform the build using `sid` or `buster` to obtain python 3.6 and still run a distroless container image?

Thanks!

**Comment by chanseokoh** [comment]
> Is support for python 3.6 distroless images coming?

Probably not, until someone contributes their time to do the work.

**Comment by jbmcfarlin31** [comment]
> > Is support for python 3.6 distroless images coming?
> 
> Probably not, until someone contributes their time to do the work.

@chanseokoh What all is involved with creating the distroless image? Perhaps I can have a stab at it if there is sufficient documentation for it.

**Comment by chanseokoh** [comment]
Not a lot of docs, I'm afraid. You need to look at `WORKSPACE` and various `BUILD` files to understand how an image is built.

**Comment by malthe** [comment]
You can get Python 3.7 using:
```docker
FROM gcr.io/distroless/python3-debian10
```
See https://github.com/GoogleContainerTools/distroless/pull/391.

**Comment by chanseokoh** [comment]
Oh, I didn't realized `gcr.io/distroless/python3-debian10` is already using Python 3.7. Thanks for pointing it out, @malthe

**Comment by jbmcfarlin31** [comment]
@malthe thanks for pointing that out! I also figured out my problem... I was using `pyinstaller` to build my executable, and it turns out that it was using `python3` on my machine which was pointing to `Python 3.5.2` - and when my executable tried running, it was breaking because my libraries require Python 3.6+.

Well, I updated my python on my box from `3.5.2` to `3.6.10` and re-ran the `pyinstaller` command, and it ended up allowing it to work with the `gcr.io/distroless/python3-debian10` image.

Thank you all for your help!
